src/main/java/com/thebois/models/beings/roles/IdleRole.java

```diff
@@ -1,13 +1,49 @@
 package com.thebois.models.beings.roles;
 
+import java.util.Collection;
+import java.util.List;
+
+import com.thebois.models.beings.IActionPerformer;
+import com.thebois.models.beings.actions.ActionFactory;
+import com.thebois.models.beings.actions.IAction;
+import com.thebois.models.beings.actions.IActionSource;
+import com.thebois.models.world.ITile;
+import com.thebois.models.world.IWorld;
+
 /**
  * The builder constructs buildings.
  */
 class IdleRole extends AbstractRole {
 
+    /**
+     * The radius at which random positions to move to are picked, in tiles.
+     */
+    private static final int RANDOM_MOVE_RADIUS = 5;
```

Comment:
Doesn't seem so random to me

Reply:
True, the radius itself isn't what's random. It's the radius in which random movement locations are picked. Do you have any suggestions for other names?